### PR TITLE
Docs gas example

### PR DESCRIPTION
**Pull request by Adamj1232**
### Description
Add more examples to the Gas modules along with a gas comparison visualization to compare against RPC gas pricing

Screen Shot 2022-10-24 at 1 07 49 PM

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] I have run `yarn check` & `yarn build` to confirm there are not any associated errors
- [x] This PR passes the Circle CI checks

### CODE REVIEW

File: docs/tailwind.config.cjs [hunk 1]
```diff
@@ -11,7 +11,20 @@ module.exports = {
...
+        conf: {
+          99: '#5aea98',
+          95: '#5dea5a',
+          90: '#bcea5a',
+          80: '#ffe600',
+          70: '#eab05a'
+        }
+      }
+    }
+  }
```
Comment (gesquinca) [inline docs/tailwind.config.cjs:27]: @Adamj1232 What are these colors for?

Comment (Adamj1232) [reply docs/tailwind.config.cjs:27]: Those colors are for the gradient on the gas page

Comment (gesquinca) [reply docs/tailwind.config.cjs:27]: Can we scope them down to just the gas component? If we're only using them there, I'd rather not add them to global theme.

File: docs/src/lib/components/gas/Gas.svelte [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    estimatedTransactionCount: null,
+    seconds: null
+  }
+</script>
+
+<div class="Gas px-6  p-4">
+  <div class="flex whitespace-nowrap mb-3 text-sm select-none">
+    <span class="flex items-center">MORE LIKELY</span>
+    <span
+      class="bg-gradient-to-r from-conf-99 via-conf-95 via-conf-90 via-conf-80 to-conf-70 h-[1px] mx-2 my-3 w-full rounded-full"
```
Comment (Adamj1232) [inline docs/src/lib/components/gas/Gas.svelte:71]: @gesquinca Usage of gradient colors

File: docs/tailwind.config.cjs (hunk 1 above)
Comment (Adamj1232) [reply docs/tailwind.config.cjs:27]: @gesquinca they are pretty specific variables and would be required to be he to use the tailwinds implementation of gradients. This pattern follows what we are doing in other projects(Gas extensions). I tagged you in the implementation.

Comment (Adamj1232) [reply docs/tailwind.config.cjs]: All cleaned up!

**Review by gesquinca — APPROVED** [review]

**Pull request merged**